pulsar-functions/runtime/src/main/java/org/apache/pulsar/functions/runtime/process/ProcessRuntime.java

```diff
@@ -106,6 +106,10 @@
         switch (instanceConfig.getFunctionDetails().getRuntime()) {
             case JAVA:
                 logConfigFile = "java_instance_log4j2.xml";
+                String palsarHome = System.getenv("PULSAR_HOME");
+                if(palsarHome!=null&&!palsarHome.trim().isEmpty()){
```

Comment:
- I believe this check is redundant since `PULSAR_HOME` is set in [bin/pulsar](https://github.com/apache/pulsar/blob/313390d86f7ff81124f75181bdd5bbd2b2be7fca/bin/pulsar#L22)
- Instead of having a hardcoded path in the `conf` folder, can we read from a separate environment variable, e.g `PULSAR_FUNCTIONS_LOG_CONF`?
Here is how it works already for broker logs with [`PULSAR_LOG_CONF`](https://github.com/apache/pulsar/blob/313390d86f7ff81124f75181bdd5bbd2b2be7fca/bin/pulsar#L160).